File: Java/백준/Bronze/1037. 약수/약수.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int N = in.nextInt();
        int max = 0;
        int min = 10000;
        while(N>0){
            int Q = in.nextInt();
            max = Q>max?Q:max;
            min = Q<min?Q:min;
            N=N-1;
        }
        System.out.println(max*min);
    }
}

File: C#/백준/Bronze/1037. 약수/약수.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            string[] aa = Console.ReadLine().Split();
            int[] b = new int[a];
            for (int i =0; i<a; i++){
                b[i] = int.Parse(aa[i]);
            }
            Console.WriteLine(b.Max()*b.Min());
        }
    }
}